Controllers/Backend/ViewSnapshots.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use FroshViewSnapshots\Services\Diff;
+use Shopware\Components\Api\Exception\ParameterMissingException;
```

Comment:
I wouldn't recommend using `ParameterMissingException` here as it is part of the API bundle. Shopware usually throws an `Enlight_Controller_Exception` when required action parameters are missing.

Reply:
Ah I see. I will amend this.